client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	debug = require( 'debug' )( 'calypso:my-sites:people:team-list' ),
+	omit = require( 'lodash/omit' );
+
+/**
+ * Internal dependencies
+ */
+var Card = require( 'components/card' ),
+	PeopleListItem = require( 'my-sites/people/people-list-item' ),
+	UsersActions = require( 'lib/users/actions' ),
+	InfiniteList = require( 'components/infinite-list' ),
+	deterministicStringify = require( 'lib/deterministic-stringify' ),
+	NoResults = require( 'my-sites/no-results' ),
+	analytics = require( 'lib/analytics' ),
+	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ),
+	config = require( 'config' );
+
+module.exports = React.createClass( {
+	displayName: 'Team',
+
+	getInitialState: function() {
+		return {
+			bulkEditing: false
+		};
+	},
+
+	isLastPage() {
+		return this.props.totalUsers <= this.props.users.length + this.props.excludedUsers.length;
+	},
+
+	fetchHasCompleted() {
+		return this.props.fetchInitialized && ! this.props.fetchingUsers;
+	},
+
+	render: function() {
+		var key = deterministicStringify( omit( this.props.fetchOptions, [ 'number', 'offset' ] ) ),
+			headerText = this.translate( 'Team', { context: 'A navigation label.' } ),
```

Comment:
also check out our translation wrapper in `i18n-calypso`, which provides `localize()` to wrap the component and will feed in `translate` as a prop

```js
class Team extends Component {
	static propTypes = {
		translate: PropTypes.func,
	};

	static defaultProps = {
		translate: identity,
	};

	doSomething() {
		const { translate } = this.props;

		return translate( 'Team', { context: 'noun', comment: 'navigation label' } );
	}
}

export default localize( Team );
```